Scripts/aws_stack_provision.ps1

```diff
@@ -7,12 +7,12 @@ param (
 
 $EXISTING_INSTANCE_COUNT = (((Get-EC2Instance -Filter @{ Name="tag:LansaVersion"; Values=$lansa_version }).Instances).InstanceId).count
 
-if ($EXISTING_INSTANCE_COUNT -eq 1 ){ 
+if ($EXISTING_INSTANCE_COUNT -eq 1 ){
   echo "VM with same lansa Version tag exist"
   echo "Checking VM Status"
```

Comment:
Why are you still using 'echo'! Stop it!. Replace with Write-Host now!

And I like the messages you are logging :)